notebooks/ivf_flat_example.ipynb

```diff
@@ -0,0 +1,803 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #DATASET_URL = "http://ann-benchmarks.com/glove-100-angular.hdf5"
</pre><p>Since we're duplicating this also in the ivf-pq notebook, can we also create a <code>load_dataset()</code> in the <code>common.py</code>?Please also note that our benchmarks python package also contains a python script for both downloading and loading the million-scale datasets from ann-benchmarks and the code might be useful to place in a more general location (such as <code>pylibraft</code>,perhaps?).   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Added <code>load_dataset()</code> to the utils. Yes, I have looked into the benchmarks python package as well. The actual download code that we can use from there  is only a <a href="https://github.com/rapidsai/raft/blob/6bbcf1fef7b60de2f85f055fb1acba7d27515b67/python/raft-ann-bench/src/raft-ann-bench/get_dataset/__main__.py#L28" target="_blank">few lines</a>. I prefer to keep a separate copy in the notebooks folder.</p>